definitions/npm/react-modal_v3.11.x/flow_v0.104.x-/react-modal_v3.11.x.js

```diff
@@ -1,6 +1,6 @@
 declare module 'react-modal' {
   declare type ModalProps = {
-    isOpen?: boolean,
+    isOpen: boolean,
```

Comment:
It's a `defaultProp`: https://github.com/reactjs/react-modal/blob/master/src/components/Modal.js#L79

Reply:
True, I was matching it up with https://github.com/reactjs/react-modal/blob/master/src/components/Modal.js#L31 and https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-modal/index.d.ts#L56 which both specify as required.

It's probably not necessary then since user doesn't get runtime proptype errors if using `.defaultProps`.

I'm thinking this PR probably isn't necessary and I may close, thoughts?